src/fundus/publishers/us/the_nation_parser.py

```diff
@@ -63,8 +83,13 @@ def topics(self) -> List[str]:
     class V2(V1):
```

Comment:
It seems they (mostly) updated the way they assign topics. It seems to me that they stopped using the `keywords` tag, and switched to `sailthru.tags`. For this article for example, no topics were parsed: https://www.thenation.com/article/archive/gop-threatens-pepfar-hiv-aids-africa/

Reply:
Thanks for catching this 👍 i altered topic extraction for `V2` accordingly